### ISSUE TITLE
Добавление реализации RS256

### ISSUE THREAD

**Issue by malikov-pro** [pintov/1c-jwt#2]
Реализовал JWT с RSA через криптографию windows, если есть желание можно добавить в этот проект. Есть реализация кодом 1С, но с автором к договоренности не пришли.

**Comment by pintov** [comment]
Да, давайте добавим. Можно пулл реквест, можно просто прислать код. Как вам удобно.

**Comment by inzarubin80** [comment]
> Реализовал JWT с RSA через криптографию windows, если есть желание можно добавить в этот проект. Есть реализация кодом 1С, но с автором к договоренности не пришли.

 Есть реализация кодом 1С - можете поделиться?

**Comment by pintov** [comment]
> > Реализовал JWT с RSA через криптографию windows, если есть желание можно добавить в этот проект. Есть реализация кодом 1С, но с автором к договоренности не пришли.
> 
> Есть реализация кодом 1С - можете поделиться?

@inzarubin80, к сожалению, в настоящий моемент в платформе 8.3.19 нет возможности сгенерировать хэш SHA256-RSA встроенными средствами.
